mcs/class/System.Windows.Forms/System.Windows.Forms/CheckedListBox.cs

```diff
@@ -423,7 +423,18 @@ public int Add (object item, CheckState check)
 				owner.UpdateCollections ();
 				return idx;
 			}
-		}
+
+			public override void Clear ()
+			{
+				owner.check_states.Clear ();
+				base.Clear ();
+			}
+
+			internal override void UpdateSelection (int removed_index)
+			{
+				owner.check_states.Remove (this[removed_index]);
```

Comment:
don't you need to call `base.UpdateSelection (removed_index)` ?

Reply:
Eh, of course. I was sure I pushed the version where the line was present.

Reply:
Weird. One `git push --force` later and it's there.